docs/onboarding/02-digital-twin-definition-language.md

```diff
@@ -4,7 +4,7 @@ DTDL is the schema by which models are described. Its important to understand ho
 
 - [Concepts](https://docs.microsoft.com/en-us/azure/digital-twins/concepts-models) - ~11 min. Covers basic concepts of DTDL language
 - [Marker tags](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-use-tags) ~10 min. Covers how to define tags, how to provision tag markers to your twins and how to query filtering by tags.
-- [Model Management](https://docs.microsoft.com/en-us/azure/digital-twins/how-to-manage-model) - ~11 min. Covers how to deploy models to ADT including creating, updates, and removal.
+
 
 > Model management is especially important for understanding what a CI/CD pipeline for ADT models might need to include.

```

Comment:
Should we move this note along with the model management link?

Reply:
fixed